packages/cli/src/webhooks/webhook-helpers.ts

```diff
@@ -213,22 +204,11 @@ export async function executeWebhook(
 		// if `Webhook` or `Wait` node, and binaryData is enabled, skip pre-parse the request-body
 		// always falsy for versions higher than 1
 		if (!binaryData) {
-			const { contentType, encoding } = req;
+			const { contentType } = req;
 			if (contentType === 'multipart/form-data') {
 				const { formDataFileSizeMax } = Container.get(GlobalConfig).endpoints;
-				const form = formidable({
-					multiples: true,
-					encoding: encoding as formidable.BufferEncoding,
-					maxFileSize: formDataFileSizeMax * 1024 * 1024,
-					// TODO: pass a custom `fileWriteStreamHandler` to create binary data files directly
-				});
-				req.body = await new Promise((resolve) => {
-					form.parse(req, async (_err, data, files) => {
-						normalizeFormData(data);
-						normalizeFormData(files);
-						resolve({ data, files });
-					});
-				});
+
+				req.body = await createMultiFormDataParser(formDataFileSizeMax)(req);
```

Comment:
maybe split this into two lines for better readability, and to reuse the parser function, instead of creating one for every request.

```suggestion
const formDataParser = createMultiFormDataParser(formDataFileSizeMax);
        
        .....
        .....
				req.body = await formDataParser(req);
```